datastructures/hashmap/hashtable.go

```diff
@@ -0,0 +1,17 @@
+package hashmap
```

Comment:
Please add a description of the package in a comment immediately proceeding the package declaration

```suggestion
Package hashmap <desciption goes here>
package hashmap
```